pages/api/authenticate.js

```diff
@@ -0,0 +1,38 @@
+import { ThirdwebSDK } from "@thirdweb-dev/sdk";
+
+const authenticate = async (req, res) => {

```

Comment:
This is the endpoint you could hit on page load/whenever you want to check if user is still logged in. Basically if it doesn't return `ok` HTTP status then user isn't logged in.

Reply:
included this in latest commit